timelock-agent/src/main/java/com/palantir/timelock/paxos/TimeLockAgent.java

```diff
@@ -377,6 +381,8 @@ private void createAndRegisterResources() {
             registrar.accept(
                     MultiClientConjureTimelockResource.jersey(redirectRetryTargeter, asyncTimelockServiceGetter));
             NonConjureTimelockResources.createJerseyResources(namespaces).forEach(registrar);
+            registrar.accept(
+                    ConjureLockWatchDiagnosticsResource.jersey(redirectRetryTargeter, asyncTimelockServiceGetter));
```

Comment:
We run some tests in AtlasDB with non-undertow. This is why I am adding a Jersey resource. In production, this should not be hit at all.